docker-compose.yml

```diff
@@ -0,0 +1,64 @@
+version: "3.9"
+
+services:
+  app:
```

Comment:
All of the service containers should have `name:` keys defined.

This is because docker compose will name containers based on the folder name.
So using `opendorme_app_1` in your make file assumes people clone the repo into `opendorme` folder.
However in my case I cloned it into `opendor` since I first dev'd it using valet.

So using:
```
    container_name: opendorme_app
```

Could mean the make file could simply connect to `opendorme_app` container instead of `opendorme_app_1`.

Reply:
I've sent a PR to address some of these here: https://github.com/bhushan/opendor.me/pull/1

Reply:
First time I see a PR to a PR in the open-source wilderness. 😂
Love how you all work hand in hand! 🎉🔥

Reply:
in previous iteration we had name, I removed it.. when we add key then it will use that name but when we dont specify it docker itself assigns names which is `opendorme_app_1`, in my experience at-least i didnt liked to give names, I had some issues with it long time back I cant recall that but from that time i stoped giving container names.

Reply:
Without giving the containers defined names - then you cannot assume you know the container name in your makefile.

How would you address that - just force everyone to clone the repo into a uniform folder name? Seems like a pretty brittle setup - since it has an essentially undocumented requirement on the parent folder name.

I would agree that giving a container a name - in some contexts - is an "anti-pattern" for Docker based ops. However it completely fits our objective here which is "Give every contributor as uniform of a local Dev as possible".

Reply:
Somewhat i agree because I couldn't recall the problem.. lets keep names for now .. please update PR to remove other changes i will merge it @mallardduck

Reply:
kk. I still find value in using stderr over logs. But if @Gummibeer  is fine with real log files that's fine.
I've updated the PR to only adjust the container names and make file.

Reply:
also @Gummibeer you've seen me do this before for sure - but it may just be a trend I'm starting.
IIRC I did the same with @nagi1 when you asked if I could help with docs on https://github.com/spatie/laravel-activitylog 😄

Reply:
@mallardduck thnx for PR 🤘🏻